src/common/geometry.js

```diff
@@ -365,6 +365,13 @@ export const downsample = (vertices, tolerance = 0.0001) => {
   return result
 }
 
+// Normalize the theta value to the range [0, 2*pi)
+export const normalizeTheta = (theta) => {
+  const TWO_PI = Math.PI * 2
+
+  return ((theta % TWO_PI) + TWO_PI) % TWO_PI
```

Comment:
I'm guessing this slightly different form has to do with an edge case near zero, but slightly negative?

Reply:
@jeffeb3 Thanks for the clarification on this. I'll revert all normalization logic.